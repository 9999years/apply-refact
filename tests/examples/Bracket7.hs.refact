[("tests/examples/Bracket7.hs:1:10-14: Suggestion: Redundant bracket\nFound:\n  if (f x) then y else z\nPerhaps:\n  if f x then y else z\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 10, endLine = 1, endCol = 15}, subts = [("x",SrcSpan {startLine = 1, startCol = 11, endLine = 1, endCol = 14})], orig = "x"}])]
