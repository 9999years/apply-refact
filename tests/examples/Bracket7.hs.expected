yes = if f x then y else z
